README.md

```diff
@@ -1,80 +1,161 @@
 <p align="center">
-  <h3 align="center">sklearn_benchmarks</h3>
+  <h1 align="center">sklearn_benchmarks</h1>
 
   <p align="center">
-    A comparative benchmarking tool for scikit-learn's estimators
+    A comparative benchmarking library for scikit-learn's estimators
     <br />
   </p>
 </p>
 
-## Table of content
 
-<ol>
-  <li><a href="#about-the-project">About</a></li>
-  <li><a href="#getting-started">Getting Started</a></li>
-  <li><a href="#usage">Usage</a></li>
-</ol>
+# About
 
-## About
+sklearn_benchmarks is a library to benchmark [scikit-learn](https://github.com/scikit-learn/scikit-learn)'s estimators against concurrent implementations. It can be used with a Python API or through a command line as described below.
 
-sklearn_benchmarks is a framework to benchmark [scikit-learn](https://github.com/scikit-learn/scikit-learn)'s estimators against concurrent implementations.
+See benchmark results [here](https://mbatoul.github.io/sklearn_benchmarks/).
 
-To consult benchmark results, see notebooks [scikit_learn_intelex_vs_scikit_learn.ipynb](https://github.com/mbatoul/sklearn_benchmarks/blob/master/scikit_learn_intelex_vs_scikit_learn.ipynb), [onnx_vs_scikit_learn.ipynb](https://github.com/mbatoul/sklearn_benchmarks/blob/master/onnx_vs_scikit_learn.ipynb) and [gradient_boosting.ipynb](https://github.com/mbatoul/sklearn_benchmarks/blob/master/gradient_boosting.ipynb).
+# Install
 
-For information regarding the benchmark environment and execution, see notebook [index.ipynb](https://github.com/mbatoul/sklearn_benchmarks/blob/master/index.ipynb).
+In order to setup the environment, you need to have `conda` installed. See instructions [here](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
 
-These notebooks automatically deployed to `github-pages`. See current results [here](https://mbatoul.github.io/sklearn_benchmarks/).
+To get a local copy up and running follow these simple example steps:
 
-sklearn_benchmarks is used through a command line as described below.
+```sh
+$ git clone -b master --single-branch https://github.com/mbatoul/sklearn_benchmarks
+$ cd sklearn_benchmarks
+$ conda env create --file dev_environment.yml
```

Comment:
Instead of calling it `dev_environment.yml` it would make more sense to call it `bench_environment.yml`. Otherwise it's confusing the readers' own dev environment.

Some CI scripts might need to be adjusted accordingly